packages/store/src/gateway/AUTO_GENERATED/auth.ts

```diff
@@ -8,15 +8,12 @@ const injectedRtkApi = api
     endpoints: (build) => ({
       authGetNonceV1: build.query<AuthGetNonceV1ApiResponse, AuthGetNonceV1ApiArg>({
         query: () => ({ url: `/v1/auth/nonce` }),
-        providesTags: ['auth'],
```

Comment:
How did you remove them @usame-algan ? This is auto-generated. We should not modify it directly

Reply:
This was done for testing only. Unfortunately not the reason why it fails for Michael so no need to merge this.